src/test/java/com/spotify/github/v3/clients/OrganisationClientTest.java

```diff
@@ -78,4 +82,34 @@ public void testAppClient() throws Exception {
     assertThat(installation.id(), is(1));
     assertThat(installation.account().login(), is("github"));
   }
+
+  @Test
+  public void getOrgMembership() throws Exception {
+    final CompletableFuture<OrgMembership> fixture =
+        completedFuture(json.fromJson(getFixture("org_membership.json"), OrgMembership.class));
+    when(github.request("/orgs/github/memberships/octocat", OrgMembership.class))
+        .thenReturn(fixture);
+    final OrgMembership orgMembership = organisationClient.getOrgMembership("octocat").get();
+    assertThat(
+        orgMembership.url().toString(), is("https://api.github.com/orgs/github/memberships/octocat"));
+    assertThat(orgMembership.role(), is("member"));
+    assertThat(orgMembership.state(), is("active"));
+  }
+
+  @Test
+  public void updateMembership() throws Exception {
+    final OrgMembershipCreate orgMembershipCreateRequest =
+        json.fromJson(getFixture("membership_update.json"), OrgMembershipCreate.class);
```

Comment:
I don't see this fixture included in resources. Does it already exist?

Reply:
Yes this already exists, both the team memberships and org memberships have the same request schema so it didn't seem worth it to create a new file with the same content. They have different responses though, so that's been reflect with the new file.

Reply:
Thanks for clarifying.